jpo-asn-runtime/src/main/java/us/dot/its/jpo/asn/runtime/serialization/OpenTypeDeserializer.java

```diff
@@ -9,6 +9,7 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.fasterxml.jackson.databind.node.ArrayNode;

```

Comment:
question: this doesn't appear to be used, do we need it?

Reply:
Good catch, removed the unused import.